// option java_multiple_files = true;
// option java_package = "io.grpc.examples.helloworld";
// option java_outer_classname = "HelloWorldProto";
// option objc_class_prefix = "HLW";

syntax = "proto3";

package clipper.grpc;

// A prediction server
service Predict {
    // Method to send a prediction request with float data
    // and receive a response
    rpc PredictFloats(PredictRequest) returns (PredictResponse) {}
}

// A prediction request message
message PredictRequest {
    repeated bytes inputs = 1;
    repeated int32 msg_ids = 2;
}

// The request message containing the user's name.
message PredictResponse {
    repeated int32 msg_ids = 1;
}
